vue-cli-2 app + quasar UI with Cypress 10

robertguss opened the pull request.

robertguss committed:
created a new vue-cli-2 app with quasar UI and Cypress 10

cowboy commented on vue-cli-2-quasar-ui/cypress.config.js:
```diff
@@ -0,0 +1,15 @@
+
+module.exports = defineConfig({
+  component: {
+    devServer(cypressDevServerConfig) {
```
FYI this isn't the correct way to specify the `devServer` config option. Please see the [Changes in 10.0.0](https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#cfd68e3db8474d6faec70e47870fbd19) doc!

kevinold replied:
I've updated

Commit:
add "# Component Testing Example:" to readme so quasar ui example is run in CI

Commit:
upgrade to latest v10

Commit:
Update to latest devServer config

Per https://www.notion.so/cypressdx/Changes-for-10-0-0-1dca213837b84d6ea297c2d12c615f3a#cfd68e3db8474d6faec70e47870fbd19

Commit:
update to latest v10 binary

Commit:
bump dependencies

Commit:
update to latest v10 binary

robertguss committed:
installed cypress eslint plugin

ZachJW34 committed:
feat: add custom mount command to mount quasar components

robertguss committed:
Merge pull request #93 from cypress-io/zachw/vue-quasar-custom-mount

created a new vue-cli-2 app with quasar UI and Cypress 10

Commit:
update v10 binary
